pkgs/by-name/tr/tre/package.nix

```
@@ -1,30 +1,57 @@
 {
   lib,
   stdenv,
-  fetchurl,
-  fetchpatch,
+  fetchFromGitHub,
+  autoreconfHook,
+  automake,
+  darwin,
+  gettext,
+  libiconv,
+  libtool,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "tre";
-  version = "0.8.0";
-  src = fetchurl {
-    url = "https://laurikari.net/tre/${pname}-${version}.tar.gz";
-    sha256 = "1pd7qsa7vc3ybdc6h2gr4pm9inypjysf92kab945gg4qa6jp11my";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "laurikari";
+    repo = "tre";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
This is the new preferred argument for fetching tags. `rev` should only be used for commits.